core/modules/query/ValueExpr.cc

```diff
@@ -271,17 +271,17 @@ std::ostream& operator<<(std::ostream& os, ValueExpr const& ve) {
 }
 
 std::ostream& operator<<(std::ostream& os, ValueExpr const* ve) {
-    if(!ve) return os << "<NULL>";
+    if (!ve) return os << "<NULL>";
     return os << *ve;
 }
```

Comment:
But this case is just a nullptr check, isn't it?  It's a very minor concern either way, ha!